source/appModules/devenv.py

```diff
@@ -102,11 +114,25 @@ def chooseNVDAObjectOverlayClasses(self, obj, clsList):
 				pass
 			clsList.insert(0, VsTextEditPane)
 
-		if ((vsMajor == 15 and vsMinor >= 3)
-			or vsMajor >= 16):
+		if (
+			(
+				self.visualStudioVersion[0] == 15
+				and self.visualStudioVersion[1] >= 3
+			)
+			or self.visualStudioVersion[0] >= 16):
 			if obj.role == controlTypes.ROLE_TREEVIEWITEM and obj.windowClassName == "LiteTreeView32":
 				clsList.insert(0, ObjectsTreeItem)
 
+			try:
+				if (
+					obj.role == controlTypes.ROLE_STATICTEXT
+					and isinstance(obj, UIA.UIA)
+					and obj.previous.previous.UIAElement.CachedAutomationId in self.INTELLISENSE_LIST_AUTOMATIONIDS

```

Comment:
```suggestion
					and obj.previous
					and obj.previous.previous
					and obj.previous.previous.UIAElement.CachedAutomationId in self.INTELLISENSE_LIST_AUTOMATIONIDS
```

Unfortunately, Python has no such way as in C# to use something like
```
					and obj.previous?.previous?.UIAElement?.CachedAutomationId in self.INTELLISENSE_LIST_AUTOMATIONIDS
```

Reply:
Thanks @leonardder, this makes sense.

Cheers,